### ISSUE TITLE
Enums in prisma generated file are not being picked up

### ISSUE THREAD

**Issue by herkulano** [prisma/graphqlgen#197]
## Description
Enums in prisma generated file are not being picked up. My understanding is that it should also pick them from the prisma generated files and generate them for the resolvers as typescript enums.

## Steps to reproduce
Add a `PostType` enum to the example https://github.com/prisma/prisma-examples/tree/master/typescript-graphql-auth

`./prisma/enums.graphql`
```graphql
enum PostType {
  MESSAGE
  ANNOUNCEMENT
  RANDOM
}
```

`./prisma/prisma.yml`
```yml
endpoint: http://localhost:4466
datamodel: 
  - datamodel.prisma
  - enums.graphql
generate:
    - generator: typescript-client
      output: ../src/generated/prisma-client/
```

Generate prisma files
```
$ prisma generate
````

Import enum in `schema.graphql`
```graphql
# import * from '../prisma/enums.graphql'
```

Run 
```
$ graphqlgen
```

## Expected results
Resolvers and enums are generated without errors

## Actual results
```
❌ Some types from your application schema have model definitions that are not defined yet

Step 1: Copy/paste the model definitions below to your application

export interface PostType {

}


Step 2: Reference the model definitions in your graphqlgen.yml file

models:
  files:
    - ./path/to/your/file.ts

Step 3: Re-run `graphqlgen`
```

## Versions
- graphqlgen: 0.2.12
- OS name and version: MacOS 10.14

**Comment by Weakky** [comment]
Hey there,

Could you please provide us your `graphqlgen.yml` configuration file?

Thanks

**Comment by arielschiavoni** [comment]
I am also having the same problem with enums. More info [here](https://github.com/prisma/graphqlgen/issues/213#issuecomment-434064121)

**Comment by schickling** [comment]
This will be fixed with #244
